app/controllers/account/tasks_controller.rb

```diff
@@ -17,6 +17,7 @@ def create
     @task = @project.tasks.build(tasks_params)
 
     if @task.save
+      @task.watchers << current_user if !@task.section?
```

Comment:
create method add_watcher(user)

Reply:
it will be updated after https://github.com/ITAR089IF/project-management-tool/pull/50